README.md

```diff
@@ -1068,6 +1068,16 @@ See also [About Rust’s Machine Learning Community](https://medium.com/@autumn_
   * [arrayfire/arrayfire-rust](https://github.com/arrayfire/arrayfire-rust) — [Arrayfire](https://github.com/arrayfire) bindings
   * [autumnai/collenchyma](https://github.com/autumnai/collenchyma) — An extensible, pluggable, backend-agnostic framework for parallel, high-performance computations on CUDA, OpenCL and common host CPU.
   * [luqmana/rust-opencl](https://github.com/luqmana/rust-opencl) — [OpenCL](https://www.khronos.org/opencl/) bindings
+* Russell Lab
+  * [cpmech/russell](https://github.com/cpmech/russell) [[russell_lab](https://crates.io/crates/russell_lab)] — Rust Scientific Library: Laboratory with special math functions, linear algebra, interpolation, quadrature, num derivation
+* Russell Ode
+  * [cpmech/russell](https://github.com/cpmech/russell) [[russell_ode](https://crates.io/crates/russell_ode)] — Rust Scientific Library: Solvers for ordinary differential equations and differential algebraic equations
+* Russell Sparse
+  * [cpmech/russell](https://github.com/cpmech/russell) [[russell_sparse](https://crates.io/crates/russell_sparse)] — Rust Scientific Library: Solvers for large sparse linear systems (wraps MUMPS and UMFPACK)
+* Russell Stat
+  * [cpmech/russell](https://github.com/cpmech/russell) [[russell_stat](https://crates.io/crates/russell_stat)] — Rust Scientific Library: Statistics calculations and (engineering) probability distributions
+* Russell Tensor
+  * [cpmech/russell](https://github.com/cpmech/russell) [[russell_tensor](https://crates.io/crates/russell_tensor)] — Rust Scientific Library: Tensor analysis, calculus, and functions for continuum mechanics
```

Comment:
Can you re-arrange this a bit? That's a pretty big wall of text for one project! Possibly just list it as one line noting the general scientific library, and then folks can follow the link to find out about the crates.

Reply:
Sure; no worries